client/src/pages/EditProfile.jsx

```diff
@@ -28,53 +31,77 @@ export default function EditProfile() {
   };
 
   return (
-    <Form method="put">
-      <label>
-        First Name:
-        <input
-          type="text"
-          name="firstname"
-          value={formData.firstname}
-          onChange={handleChange}
-        />
-      </label>
-      <label>
-        Last Name:
-        <input
-          type="text"
-          name="lastname"
-          value={formData.lastname}
-          onChange={handleChange}
-        />
-      </label>
-      <label>
-        Email:
-        <input
-          type="email"
-          name="email"
-          value={formData.email}
-          onChange={handleChange}
-        />
-      </label>
-      <label>
-        City:
-        <input
-          type="text"
-          name="city"
-          value={formData.city}
-          onChange={handleChange}
-        />
-      </label>
-      <label>
-        Image URL:
-        <input
-          type="text"
-          name="image"
-          value={formData.image}
-          onChange={handleChange}
-        />
-      </label>
-      <button type="submit">Save</button>
+    <Form method="put" className="edit-form">
+      <section className="image-edit-container">
+        <div className="photoComponent">
+          {user.image !== null ? (
+            <img src={user.image} alt="user profile" className="profilePhoto" />
+          ) : (
+            <FaRegUserCircle className="noPhoto" />
+          )}
+        </div>
+
+        <div className="img-input">
+          <label className="edit-label">
+            <p>Image URL:</p>
```

Comment:
Using a `<p>` tag within a `<label>` tag is not considered correct HTML practice. The `<label>` element is intended to associate a form control with a label, and it should contain inline elements directly associated with the form control. A `<p>` tag is a block-level element and is generally used for paragraphs of text, not for labeling form inputs.